ocaml/idl/datamodel_errors.ml

```diff
@@ -1141,7 +1141,10 @@ let _ =
   error Api_errors.invalid_cluster_stack [ "cluster_stack" ]
     ~doc:"The cluster stack provided is not supported." ();
   error Api_errors.pif_not_attached_to_host [ "pif"; "host" ]
-    ~doc:"Cluster_host creation failed as the PIF provided is not attached to the host." ()
+    ~doc:"Cluster_host creation failed as the PIF provided is not attached to the host." ();
+  error Api_errors.clustering_operation_in_progress []
```

Comment:
Should we include a clustering operation string here?

Reply:
Can you reuse other_operation_in_progress?

Reply:
Yes, but then I need to provide a ref to the cluster_host in use. Should I just include a specific error string in there instead, since blocking a PIF operation with "another clustering operation in progress"  might be confusing?

Reply:
e.g. `["Cluster_host"; "Cannot allow unplug on any PIF if enabling or creating a cluster host on any PIF"]`

Reply:
` error Api_errors.other_operation_in_progress ["class"; "object"]` you need to give a ref as the 2nd argument to `other_operation_in_progress`, not an arbitrary string.

Reply:
Should I just specify the PIF? I think that would be confusing for a user. We don't access the specific cluster_host in this, otherwise we could just simply block allowing unplug for that specific PIF.

Reply:
Unless we can get it from the task list?

Reply:
Yeah just provide the cluster ref.

Reply:
Cool, done. Is this good to merge?

Reply:
I think you can drop `clustering_operation_in_progress`, you are not using it anymore.

Reply:
Switched to `other_operation_in_progress`